src/insights_client/__init__.py

```diff
@@ -38,30 +38,21 @@
 logger = logging.getLogger(__name__)
 
 
-def client_debug(message):
-    """Debug a log message when logging isn't available yet.
-
-    Set 'INSIGHTS_DEBUG' variable to enable this method.
-
-    :param message: Text to display
-    :type message: str
-    """
-    if not INSIGHTS_DEBUG:
-        return
-
-    prefix = "insights-client debug"
-    suffix = ""
-    if NO_COLOR or not sys.stdout.isatty():
-        prefix += ":"
+def set_up_logging():
+    handler = logging.StreamHandler(sys.stdout)
+    if INSIGHTS_VERBOSE:
+        logger.setLevel(logging.DEBUG)
+        log_format = LOG_FORMAT
     else:
-        prefix = "\033[40m" + prefix + "\033[0m\033[33m"
-        suffix = "\033[0m"
-
-    print(prefix + " " + message + suffix, file=sys.stderr)
+        log_format = "%(message)s"
```

Comment:
If we know we don't want to log anything from this module, should this function just `return None` directly?

Reply:
Agreed. It makes sense to not setup stuff if we don’t want to use it. Also we’d get rid of this explicit re-stating of the default format.